tests/folder_tests/test_symlinks.py

```diff
@@ -67,7 +67,7 @@ def test_create_symlink(self):
 
         self.assertTrue(os.path.exists(self.aaa))
         self.assertTrue(os.path.exists(self.aaa_work))
-        if is_windows():
+        if not is_windows():
```

Comment:
When switching to `is_*`, this check was flipped.

Reply:
🤦‍♂